src/commands/synthetics/run-test.ts

```diff
@@ -124,6 +125,11 @@ export class RunTestCommand extends Command {
       return safeExit(1)
     }
     const {tests, overriddenTestsToTrigger, summary} = testsToTriggerResult
+    if (this.config.locations.length) {
+      overriddenTestsToTrigger.forEach((overriddenTestToTrigger) => {
+        overriddenTestToTrigger.locations = this.config.locations
+      })
+    }
```

Comment:
Sorry for the late review.
It seems most of the config override is done in (or around) `handleConfig` in `synthetics/utils`: https://github.com/DataDog/datadog-ci/blob/master/src/commands/synthetics/utils.ts#L43-L85
Out of curiosity, what is the reason for doing the `locations` override so far from the original override?